manifests/server/config.pp

```diff
@@ -130,6 +131,15 @@
     }
   }
 
+  if $jit_enable {
```

Comment:
What happens if you set jit_enable = true and then set it to false afterwards?

Reply:
Good point. I will rework it so it becomes a bool.